appveyor.yml

```diff
@@ -20,6 +20,9 @@ matrix:
     - image: Ubuntu1804
       MSYSTEM: MSVC
 
+init:
+- cmd: set PATH=C:\Python36;%PATH%
```

Comment:
why?

Reply:
The shutil.which requires at least Python 3.5. I just put the latest supported Python 3.6. 
Does Ninja officially support a minimum Python release? Then there would be a different way to do this fix depending on minimum Python version.

Reply:
Nothing really official. As Python 2.7 has worked with configure.py in the past, I think people expect that to continue to work for the time being.